rawfile/rawfile.py

```diff
@@ -26,6 +27,10 @@
 @click.option("--log-level", envvar="LOG_LEVEL", default="INFO")
 @click.option("--reserved-capacity", envvar="RESERVED_CAPACITY", default="0")
 @click.option("--capacity-override", envvar="CAPACITY_OVERRIDE", default="0")
+@click.option("--ga-enabled/--ga-disabled", is_flag=True)
+@click.option("--ga-id", envvar="GA_ID", default="G-LPD03CGEVN")
+@click.option("--ga-key", envvar="GA_KEY", default="ibj6b0nxRqC5ti4xzUgRow")
```

Comment:
These are the values to the actual openebs property: https://github.com/openebs/google-analytics-4/blob/develop/usage/const.go#L5

Does this need to be disabled CI?

Reply:
Yes I think we should disable on the CI
It's only enabled now so I can check it works, once I'm happy it's working I'll both disable it on CI and set the actual ga properties